pkg/security/util.go

```diff
@@ -2,6 +2,7 @@ package security
 
 const (
 	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
+	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"
```

Comment:
It is directed specifically at the proposed field in the `PodSecurityContext` of :
```go
 // FSGroup is a supplemental group that all containers in a pod run under.  This group will own
    // volumes that the Kubelet manages ownership for.  If this is not specified, the Kubelet will
    // not set the group ownership of any volumes.
    FSGroup *int64 `json:"supplementalGroup"`
```

It could be used as other things, perhaps a default supplemental group for the namespace if we wanted.  Rebrand as `sa.scc.default-group`?